src/pages/Comments/index.tsx

```diff
@@ -20,7 +22,7 @@ import LocalStorageUtils from "src/utils/LocalStorageUtils";
 
 const Comments = () => {
     const user = LocalStorageUtils.getUser();
-    const postId = "727624bd-a537-4d37-8429-b9b8be310e1c";
+    const postId = useParams<string>().id!;
```

Comment:
I don't think you generic this correctly, it should be an object or something right? Can you help check?

Reply:
![Screen Shot 2022-01-16 at 23 08 23](https://user-images.githubusercontent.com/59289834/149667869-4d610211-ac46-494a-9e69-3afb23642ffb.png)
I think string is the correct generic of this